uday.txt

```diff
@@ -0,0 +1,2 @@
+Hello Team, 
+This is uday, New on-boarded into the team. 
```

Comment:
I think you have missed the name. It's Uday Mittapally.

Reply:
Yeah sure , Its addressed, Thats a good catch